code/src/components/TodoList.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from "react";
+import { TodoItem } from "./TodoItem.js";
+import { TodoInput } from "./TodoInput.js";
+import { TodoSummary } from "./TodoSummary.js";
+import { useSelector } from "react-redux";
+import { RemoveAll } from './RemoveAll'
+
+export const TodoList = () => {
+  const list = useSelector((store) => store.todos.list);
```

Comment:
you are only interested in the items array, so better to get only that array from the store instead of the whole object.
```
const items = useSelector(store => store.todos.list.items);
```